src/integrationCapture.ts

```diff
@@ -0,0 +1,63 @@
+import { SDKEventCustomFlags } from './sdkRuntimeModels';
+import { Dictionary, queryStringParser, getCookies, getHref } from './utils';
+
+const integrationIdMapping: Dictionary<string> = {
+    fbclid: 'Facebook.ClickId',
+    _fbp: 'Facebook.BrowserId',
+};
+
+export default class IntegrationCapture {
+    public clickIds: Dictionary<string>;
+
+    /**
+     * Captures Integration Ids from cookies and query params and stores them in clickIds object
+     */
+    public capture(): void {
+        this.captureCookies();
+        this.captureQueryParams();
+    }
+
+    /**
+     * Captures cookies based on the integration ID mapping.
+     */
+    public captureCookies(): void {
+        const cookies = getCookies(Object.keys(integrationIdMapping));
+        this.clickIds = { ...this.clickIds, ...cookies };
```

Comment:
I assume the idea for this is that there may be multiple times in which we need to call `capture()`, and so we want to ensure we are not losing previous click ids?